cocos/renderer/backend/ProgramState.cpp

```diff
@@ -112,6 +112,9 @@ TextureInfo::TextureInfo(const std::vector<uint32_t>& _slots, const std::vector<
 TextureInfo::TextureInfo(const TextureInfo &other)
     : slot(other.slot)
     , textures(other.textures)
+#if CC_ENABLE_CACHE_TEXTURE_DATA == 1
```

Comment:
`#if CC_ENABLE_CACHE_TEXTURE_DATA` is enough

Reply:
yes